backend/Auth/ResetPassword.ts

```diff
@@ -9,6 +9,10 @@ import {
 import { sendEmail, baseUrl } from "../Utils/EmailUtils.js";
 import { makeJWT } from "../Utils/JWTUtils.js";
 import { JWTModel } from "../DB/models/JWT";
+import jwt from "jsonwebtoken";
+
+const { verify } = jwt;
+const secret = process.env.JWT_SECRET || "secret"; // JWT_SECRET="JUC2021s3cr3t"
```

Comment:
I think I'd prefer removing the default value for JWT_SECRET and also the comment that seems to potentially leak a possible value for it. The reason is that this project is open source and so if it ever gets used by people a hacker could look at this code and then think "oh i wonder if this deployed instance has set JWT_SECRET or not. If not, I know what the secret is (`"secret"`). They also would probably see the comment and think "were they silly enough to put their production env variables in a comment?

The best thing to do would probably be to create a `.env.example` file, similar to what leah has done in her not-yet-merged PR at https://github.com/JustUtahCoders/flax-crm/pull/56/files#diff-a3046da0d15a27e89f2afe639b25748a7ad4d9290af3e7b1b6c1a5533c8f0a8c. And that file could be copied by each person to be `.env` in their project, but then in production they'd set it to a different value.

```suggestion
const jwtSecret = process.env.JWT_SECRET
```

Reply:
This makes sense!  Thank you!  Leah's example was very helpful.